base/reset.css

```diff
@@ -0,0 +1,7 @@
+html, body, div, span, iframe,h1, h2, h3, a, img, ul, li, aside, canvas, details, footer, header, nav, section, audio, video{
```

Comment:
You should use the universal selector here instead of writing these tags.
`*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}`

Reply:
Thanks for the suggestions, I have made the changes in the file.

Reply:
Yes the changes look good you can merge it @mnath30

Reply:
Yes, you can merge it now. 🚀